### DISCUSSION TITLE
Docker manage - Show more metrics of running containers.

### DISCUSSION THREAD

**Discussion by davidtranjs** [Dokploy/dokploy#552] [Ideas]
Hi everyone, I love Dokploy as it is the best PaSS in my opinion, and I want to contribute, so I want pick a small feature to begin

One feature that I really want to add for now is, in the Docker manage screen `/dashboard/docker`, currently it just shows these states `name, state, status, image`, but I really need the info of `port` show in this table too

![Xnip2024-10-13_18-03-08](https://github.com/user-attachments/assets/e2103aa4-b5c4-40f3-930f-83e167e8c4f5)

Are you thinking this is a good feature to add? If yes, it would be perfect if you have any suggestions for me to contribute it, such as avoiding common issues, any conflict with feature updates?

**Comment by Siumauricio** [comment]
Hi David, if you are willing to send a PR on that that would be great!
